src/charm.py

```diff
@@ -43,7 +43,7 @@
 COMMON_DATABASE_NAME = "free5gc"
 GRPC_PORT = 9876
 WEBUI_URL_PORT = 5000
-WEBUI_SERVICE_NAME = "webui"
+SDCORE_CONFIG_SERVICE_NAME = "nms"
```

Comment:
Already discussed this with @patriciareinoso - I think the bug this PR fixes is caused by the confusion about the meaning of 'service' here. We're not talking about the K8s service but the application. 
Perhaps we could use less confusing name here? WDYT?

Reply:
we need to use `self.app.name`, otherwhise if we use another name to deploy the nms, it won't work. 
but yes, I agree, that the `service` part was confusin to me

Reply:
Yes, `self.app.name` is perfect.